bye_splits/production/prod_params.py

```
@@ -26,7 +26,8 @@
     if htcondor:
         files_photons = glob.glob('/home/llr/cms/sauvan/DATA_UPG/HGCAL/Ntuples/study_autoencoder/3_22_1/SinglePhoton_PT2to200/GammaGun_Pt2_200_PU0_HLTWinter20_std_ae_xyseed/210430_091126/ntuple*.root')
     else:
-        files_photons = [ str(Path('/data_CMS') / 'cms' / os.environ['USER'] / 'TriggerCells' / 'hadd.root') ]
+        #files_photons = [ str(Path('/data_CMS') / 'cms' / os.environ['USER'] / 'TriggerCells' / 'hadd.root') ]
+        files_photons = [ str(Path('/data_CMS') / 'cms' / 'alves' / 'TriggerCells' / 'hadd.root') ]
```

This is supposed to work even after my personal file is gone. I think I prefer the original version. As an alternative, you could throw an error message if the directory is not found; in this latter case you could use any default directory you like.

---

For that to work, the file would have to be USER/TriggerCells, but that isn't the case for an arbitrary user (for example, I don't even have a directory in /data_CMS/cms/). That's likely a problem on its own, but regardless the file should either 1) be kept in a shared location on the llr_servers so that anyone can access it or 2) created within the pipeline. Option (1) seems significantly easier for now, but you're right that it should probably be kept somewhere other than your personal folder.